### PR TITLE
Update to outdated files in the dev-1.17-ko.8.

### PR DESCRIPTION
**Pull request by ysyukr**
### 18 of 20 files to be modified
- [x] content/ko/docs/concepts/architecture/nodes.md
- [x] content/ko/docs/concepts/cluster-administration/cluster-administration-overview.md
- [x] content/ko/docs/concepts/configuration/assign-pod-node.md
- [x] content/ko/docs/concepts/containers/container-lifecycle-hooks.md
- [x] content/ko/docs/concepts/overview/working-with-objects/annotations.md
- [x] content/ko/docs/concepts/overview/working-with-objects/labels.md
- [x] content/ko/docs/concepts/overview/working-with-objects/names.md
- [x] content/ko/docs/concepts/workloads/controllers/deployment.md
- [ ] content/ko/docs/concepts/workloads/pods/init-containers.md
> #19603 에서 반영됨
- [x] content/ko/docs/home/_index.md
- [ ] content/ko/docs/reference/glossary/kube-scheduler.md
> #19712 진행시 문제가 있어 #19708 과 함께 진행됨
- [x] content/ko/docs/setup/_index.md
- [x] content/ko/docs/setup/production-environment/container-runtimes.md
- [x] content/ko/docs/tasks/administer-cluster/cluster-management.md
- [x] content/ko/docs/tasks/manage-kubernetes-objects/declarative-config.md
- [x] content/ko/examples/application/deployment.yaml
- [x] content/ko/examples/application/simple_deployment.yaml
- [x] content/ko/examples/application/update_deployment.yaml
- [x] content/ko/examples/controllers/daemonset.yaml
- [x] content/ko/examples/controllers/nginx-deployment.yaml

### 1 files to be renamed
- [x] R098 content/ko/docs/concepts/containers/container-environment-variables.md -> content/ko/docs/concepts/containers/container-environment.md

from #19859
/language ko

### CODE REVIEW

File: content/ko/docs/concepts/architecture/nodes.md [hunk 1]
```diff
@@ -180,7 +180,7 @@ kubelet은 `NodeStatus` 와 리스 오브젝트를 생성하고 업데이트 할
   보다 훨씬 길다).
 - kubelet은 10초마다 리스 오브젝트를 생성하고 업데이트 한다(기본 업데이트 주기).
   리스 업데이트는 `NodeStatus` 업데이트와는
-  독립적으로 발생한다.
+  독립적으로 발생한다. 리스 업데이트가 실패하면 kubelet에 의해 재시도 하며 7초로 제한된 지수 백오프를 200 밀리초에 시작한다.
```
Comment (gochist) [inline content/ko/docs/concepts/architecture/nodes.md]:
```suggestion
  독립적으로 발생한다. 리스 업데이트가 실패하면 kubelet에 의해 재시도하며 7초로 제한된 지수 백오프를 200 밀리초에서 부터 시작한다.
```

1. 띄어 쓰기
1. 200 밀리초부터 시작해서 7초까지 백오프한다는 의미를 보다 잘 전달하기 위해 수정해봤습니다. 참고바랍니다.

File: content/ko/docs/setup/_index.md [hunk 2]
```diff
@@ -49,63 +49,6 @@ card:
...
-| [SUSE](https://www.suse.com/)        |              | &#x2714;       |             |      |          |
-| [SysEleven](https://www.syseleven.io/)         | &#x2714;             |        |             |      |          |
-| [Tencent Cloud](https://intl.cloud.tencent.com/)  | [Tencent Kubernetes Engine](https://intl.cloud.tencent.com/product/tke) |   &#x2714;   |   &#x2714; |    |      |   &#x2714;  |
-| [VEXXHOST](https://vexxhost.com/)         | &#x2714;             | &#x2714;      |             |      |          |
-| [VMware](https://cloud.vmware.com/) | [VMware Cloud PKS](https://cloud.vmware.com/vmware-cloud-pks)              |[VMware Enterprise PKS](https://cloud.vmware.com/vmware-enterprise-pks)        |   [VMware Enterprise PKS](https://cloud.vmware.com/vmware-enterprise-pks)          | [VMware Essential PKS](https://cloud.vmware.com/vmware-essential-pks)      |          |[VMware Essential PKS](https://cloud.vmware.com/vmware-essential-pks)
-| [Z.A.R.V.I.S.](https://zarvis.ai/) | &#x2714; | | | | | |
+[쿠버네티스 인증](https://github.com/cncf/k8s-conformance/#certified-kubernetes) 공급자의 목록과 "[파트너](https://kubernetes.io/partners/#conformance)"를 참조한다.
```
Comment (gochist) [inline content/ko/docs/setup/_index.md]:
```suggestion
[공인 쿠버네티스](https://github.com/cncf/k8s-conformance/#certified-kubernetes) 공급자의 목록과 "[파트너](https://kubernetes.io/partners/#conformance)"를 참조한다.
```

'교육' 탭의 내용과 일관성을 위해 위와 같이 의견 드립니다. 참고바랍니다.

File: content/ko/docs/setup/production-environment/container-runtimes.md [hunk 3]
```diff
@@ -182,27 +182,47 @@ sysctl --system
...
+echo 'deb http://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/Raspbian_10/ /' > /etc/apt/sources.list.d/devel:kubic:libcontainers:stable.list
+wget -nv https://download.opensuse.org/repositories/devel:kubic:libcontainers:stable/Raspbian_10/Release.key -O- | sudo apt-key add -
 
 # CRI-O 설치
-apt-get install -y cri-o-1.15
+sudo apt-get install cri-o-1.17
+{{< /tab >}}
 
+{{< tab name="Ubuntu 18.04, 19.04 and 19.10" codelang="bash" >}}
+# 리포지트리 설치
```
Comment (gochist) [inline content/ko/docs/setup/production-environment/container-runtimes.md]:
```suggestion
# 리포지터리 설치
```

한글화 용어집 참고 바랍니다.

**Comment by ysyukr** [comment]
@gochist 리뷰 감사합니다. 업데이트 진행했습니다.

**Comment by gochist** [comment]
@ysyukr 감사합니다!
/lgtm
/approve

**Pull request merged**